server/channels/app/import_functions.go

```diff
@@ -189,15 +189,16 @@ func (a *App) importTeam(rctx request.CTX, data *imports.TeamImportData, dryRun
 	}
 
 	rctx.Logger().Info("Importing team", fields...)
+	teamName := strings.ToLower(*data.Name)
 
 	var team *model.Team
-	team, err := a.Srv().Store().Team().GetByName(*data.Name)
+	team, err := a.Srv().Store().Team().GetByName(teamName)
 
 	if err != nil {
 		team = &model.Team{}
 	}
 
-	team.Name = *data.Name
+	team.Name = teamName
```

Comment:
Do we really need to modify this again? We are getting the team by `teamName`, so it is given that the team name will be `teamName`.

Reply:
If the team can't be found, `team` is an empty struct. Hence, we need to set the Name.

Reply:
Ah yes, in that case, maybe we can move this inside the `if` condition?